src/components/Image/src/Image.vue

```diff
@@ -171,6 +171,12 @@ export default {
 				img.srcset = this.srcset;
 			}
 
+			// Needed to not load the full size image
+			// and match the size loaded in the UI
+			if (this.$attrs?.sizes) {
```

Comment:
i feel like we probably should just make `sizes` a prop if we're expecting it and using it inside the component